app/assets/stylesheets/welcome.scss

```diff
@@ -1,3 +1,4 @@
 // Place all the styles related to the welcome controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+$backdrop : #C9D6AE;
```

Comment:
Color `#C9D6AE` should be written as `#c9d6ae`<br>Variable names should be followed immediately by a colon